### ISSUE TITLE
[Feature request] Add unsuitable roads display 

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#8189]
For the map viewing process, while using a certain navigation profile, it may be convenient to be able to see the roads, which cannot be used with the selected profile, e.g. highlight too narrow streets for the car.

**Comment by ScriptForAll** [comment]
I agree with this feature request since it makes use of a tagging scheme that isn't very used. However I have some questions regarding this:

Firstly, how would this be implemented? 
Would it use `width=*` and compare with an user-defined car width and possibly with a tolerance threshold (I reckon that one wouldn't ride a 2 meters wide car across a 2 meters wide street)?  Would it use the tag `narrow=yes`? If both approaches were to be implemented how would they be distinguished from one another?

Secondly, how would the rendering be done? Symbols, highlights, colours, patterns?
